LayoutTests/fast/loader/stateobjects/popstate-does-not-fire-with-page-cache.html

```diff
@@ -35,13 +35,13 @@
         break;            
     case 2:
         debug('going back to page 1 with new state object');
-        setTimeout(function() {
+        timeoutHandle = setTimeout(function() {
             testWindow.history.back();
             setTimeout(() => {
                 shouldBe("testWindowPopstateFireCount", "1");
                 testWindow.close();
                 finishJSTest();
-            }, 100);
+            }, 20000);
```

Comment:
This seems excessive.  Can't we poll until testWindowPopstateFireCount is 1?  If not, could we do 2000ms?